go.mod

```diff
@@ -40,6 +40,7 @@ require (
 	github.com/ipfs/go-log/v2 v2.5.1
 	github.com/ipfs/go-merkledag v0.7.0
 	github.com/ipld/go-car v0.5.0
+	github.com/jonboulle/clockwork v0.3.1-0.20230117163003-a89700cec744
```

Comment:
Why is it untagged?

Have you seen [this](https://github.com/benbjohnson/clock)? We already indirectly depend on it, and it has a simpler API, e.g, we won't need to keep Clock and Timer fields in the queue, and only one Clock. We won't need an untagged release there as well. WDYT on switching?